.circleci/config.yml

```diff
@@ -341,6 +341,10 @@ jobs:
               # run tests with code coverage
               CORRADE_TEST_COLOR=ON GTEST_COLOR=yes ./build.sh --headless --bullet --with-cuda --run-tests --cmake-args='-DCMAKE_CXX_FLAGS="--coverage"'
 
+              #with bullet installed, run tutorial validation
+              #TODO: loop for all tutorials in the directory?
+              python examples/tutorials/lighting_tutorial.py
```

Comment:
Please see https://github.com/facebookresearch/habitat-sim/blob/master/tests/test_examples.py for how to test things like this.

Reply:
What do you think about moving `stereo_agent.py` and `new_actions.py` into `examples/tutorials` to separate stand-alone tutorial scripts from the example demo_runner?

Reply:
That is fine by me :)